app/workers/evss/disability_compensation_form/submit_form0781.rb

```diff
@@ -0,0 +1,144 @@
+# frozen_string_literal: true
+
+module EVSS
+  module DisabilityCompensationForm
+    class SubmitForm0781
+      include Sidekiq::Worker
+      include JobStatus
+
+      FORM_ID_0781 = '21-0781' # form id for PTSD
+      FORM_ID_0781A = '21-0781a' # form id for PTSD Secondary to Personal Assault
+      FORMS_METADATA = {
+        FORM_ID_0781 => { docType: 'L228' },
+        FORM_ID_0781A => { docType: 'L229' }
+      }.freeze
+
+      STATSD_KEY_PREFIX = 'worker.evss.submit_form0781'
+
+      # Sidekiq has built in exponential back-off functionality for retrys
+      # A max retry attempt of 10 will result in a run time of ~8 hours
+      # This job is invoked from 526 background job
+      RETRY = 10
+
+      sidekiq_options retry: RETRY
+
+      # This callback cannot be tested due to the limitations of `Sidekiq::Testing.fake!`
+      sidekiq_retries_exhausted do |msg, _ex|
+        log_message_to_sentry(
+          "Failed all retries on SubmitForm0781 submit, last error: #{msg['error_message']}",
+          :error
+        )
+      end
+
+      # Performs an asynchronous job for generating and submitting 0781 + 0781A PDF documents to VBMS
+      #
+      # @param auth_headers [Hash] The VAAFI headers for the user
+      # @param evss_claim_id [String] EVSS Claim id received from 526 submission
+      # @param saved_claim_id [Integer] Saved Claim id from 526 submission
+      # @param submission_id [String] The submission id of 526, uploads, and 4142 data
+      # @param form_content [Hash] The form content for 0781 + 0781A submission
+      #
+      def perform(auth_headers, evss_claim_id, saved_claim_id, submission_id, form_content)
+        with_tracking('Form0781 Submission', saved_claim_id, submission_id) do
+          @parsed_form = parse_form(form_content)
+          @parsed_form0781 = get_form_0781
+          @parsed_form0781a = get_form_0781a
+
+          # process 0781 and 0781a
+          process_0781(auth_headers, evss_claim_id, FORM_ID_0781, @parsed_form0781) if @parsed_form0781.present?
+          process_0781(auth_headers, evss_claim_id, FORM_ID_0781A, @parsed_form0781a) if @parsed_form0781a.present?
+        end
+      rescue StandardError => error
+        # Cannot move job straight to dead queue dynamically within an executing job
+        # raising error for all the exceptions as sidekiq will then move into dead queue
+        # after all retries are exhausted
+        retryable_error_handler(error)
+        raise error
+      end
+
+      private
+
+      def parse_form(form_content)
+        form_content = form_content.to_json if form_content.is_a?(Hash)
```

Comment:
The only reason i kept this check is to verify that the parameter "form" passed to the method from 526 is converted else it will fail in pdf generation

Reply:
Alright. So did make the change to remove the check and to_json assuming its already json